static/app/views/stories/useStoriesLoader.tsx

```diff
@@ -0,0 +1,62 @@
+import {useCallback, useEffect, useState} from 'react';
+
+import storiesContext from 'sentry/views/stories/storiesContext';
+import type {ResolvedStoryModule} from 'sentry/views/stories/types';
+
+interface Props {
```

Comment:
nit: I've tried not to call hook parameters `props` since that's kind of specific to components. I've been using the word `Options`

Reply:
I've written both, but I stuck with `props` because RTL has `initialProps` https://testing-library.com/docs/react-testing-library/api#initialprops

Reply:
ah yea